### ISSUE TITLE
Synchornizing from Qfield is not working

### ISSUE THREAD

**Issue by Ipickedausername** [opengisch/qfieldsync#528]
Hello, 

After collecting data with Qfield, when I try to synchronize these data to my original project, I simply cannot.

I receive this message : QFieldSync: Opened original project (with the project path). 

Since Qfieldsync is not working well since the beginning (or I cannot make it work), I used to manually extract data from the Qfield project and add it to my data. 
But now, I can't see my data in the Qfield project (but it does exist in my phone). 
Same issue with pictures. Before I could see them in the DCIM folder, but now this folder is empty (but the pictures are visible in my phone). 

Is there something wrong I do ? 

The picture can be taken and displayed on my phone: 

![image_2023-06-26_09-07-41-311](https://github.com/opengisch/qfieldsync/assets/72577399/b0f2f0be-3519-4c17-92ab-3e6a3ce1d200)

Pictures are well saved in the DCIM folder, on my phone (with a correct name): 

![image_2023-06-26_09-21-22-702](https://github.com/opengisch/qfieldsync/assets/72577399/79a0d1b4-c802-4744-996d-2fcf0c85e31b)

But doesn't exist in my phone's directories when opened on windows:

![Sans titre2](https://github.com/opengisch/qfieldsync/assets/72577399/bc787201-c64d-4b1c-b8bd-3845dbae5e95)

The project is found in my phone from Qfield, but not when I'm searching for it on my phone. 

thank you for your attention

**Comment by nirvn** [comment]
@Ipickedausername , that is super odd. Have you tried to use the 'Send compressed folder to...' action when clicking on the DCIM folder's 3-dot menu in QField?

**Comment by nirvn** [comment]
@Ipickedausername , could you try the APKs here with a project of yours: https://github.com/opengisch/QField/pull/4402 -- photos should be visible there.

**Comment by Ipickedausername** [comment]
Hello, 

> that is super odd. Have you tried to use the 'Send compressed folder to...' action when clicking on the DCIM folder's 3-dot menu in QField?

This option works to extract pictures from the phone and send them to another app in order to open them on my computer. 

> could you try the APKs here with a project of yours: https://github.com/opengisch/QField/pull/4402 -- photos should be visible there.
With this APKs, if I open my project, it is empty.

**Comment by nirvn** [comment]
@Ipickedausername , right, it'd be empty because the APK has a different app name (i.e. ch.opengis.qfield_beta instead of ch.opengis.qfield). Can you take a few dummy photos and check if you see them when connecting with your USB cable? 

When you confirm the test photos are visible, we can push an official QField 2.8.4 on the playstore, you'll be able to then see the photos in your main project.

**Comment by Ipickedausername** [comment]
I took 3 pictures. I can see them in the folder on the project:

![image_2023-06-26_15-00-05-836](https://github.com/opengisch/qfieldsync/assets/72577399/acbb7f8b-40cc-4a2d-af8f-52974001a2a5)

But empty on windows. 

![Vide](https://github.com/opengisch/qfieldsync/assets/72577399/2761547f-827f-44be-b5b0-5d2a082854ae)

Is there something I'm missing ? Before the modifications related to google's policy changes, it was quite simple to get the picture from the folder in the phone. But maybe something has changed ? 
It is like pictures are saved somewhere in Qfield, but not visible as pictures. I check the whole phone to find these pictures. Nowhere ^^'

**Comment by nirvn** [comment]
@Ipickedausername , are you sure you are looking in the right directory? From the last windows explorer screenshot, it doesn't look like you are looking inside your Galaxy S9+, i.e.:

![image](https://github.com/opengisch/qfieldsync/assets/1728657/ecd1e152-a929-49f8-bbfd-0419f41c5020)

Your original explorer screenshot showed a "proper" path here:

![image](https://github.com/opengisch/qfieldsync/assets/1728657/891b3967-2cfa-4dec-9603-cc3a73193131)

Also, do you have other folders in your project folder alongside DCIM? Can you check if photos are located somewhere else?

**Comment by Ipickedausername** [comment]
Wrong image indeed. But empty. 

![vide2](https://github.com/opengisch/qfieldsync/assets/72577399/a1d8b3bc-0e5b-45f4-9a43-e3784e9bf11e)

I used the searching option in windows for all the folders.

**Comment by nirvn** [comment]
@Ipickedausername , and you're 100% sure you are looking into ch.opengis.qfield_beta? sorry, just want to make sure our wires aren't crossing :)

**Comment by Ipickedausername** [comment]
I looked into : ch.opengis.qfield_beta
And YES, found the 3 pictures. 

There is a bug with windows/my phone. I checked twice, and I did it again because of your message. 
The folder was empty, but with an image of hard rive and windows logo. 

![imageddd](https://github.com/opengisch/qfieldsync/assets/72577399/16b81385-9421-4938-9b53-787202760b10)

**Comment by Ipickedausername** [comment]
But I still cannot synchronize it with my original project using QfieldSync. 
But I can synchronize manually, which is already ok.

**Comment by nirvn** [comment]
@Ipickedausername , by "now I can find all the projects from Qfield master-dev", you mean you can see the photos of your projects you took with QField master-dev?

**Comment by Ipickedausername** [comment]
Yes I can see them. Can copy them.

**Comment by Ipickedausername** [comment]
When I try to synchronize for the first time, I always get this message: 

![msg](https://github.com/opengisch/qfieldsync/assets/72577399/b0b270fe-b7fc-416d-aff3-1ca74488d954)

Nothing change for the project, I cannot access any data from the synchronization. 

And then, when I try again, I get this message and no possibility to trysynchronizing again: 

![lmsg2](https://github.com/opengisch/qfieldsync/assets/72577399/25776159-b3ea-4f49-b0a3-cb9a2439890f)
